2022-02-28 16:00:32,870 INFO    MainThread:626894 [wandb_setup.py:_flush():71] setting env: {}
2022-02-28 16:00:32,870 INFO    MainThread:626894 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 16:00:32,870 INFO    MainThread:626894 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 16:00:32,870 INFO    MainThread:626894 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_160032-1pealcc3/logs/debug.log
2022-02-28 16:00:32,870 INFO    MainThread:626894 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_160032-1pealcc3/logs/debug-internal.log
2022-02-28 16:00:32,870 INFO    MainThread:626894 [wandb_init.py:init():404] calling init triggers
2022-02-28 16:00:32,870 INFO    MainThread:626894 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-28 16:00:32,870 INFO    MainThread:626894 [wandb_init.py:init():460] starting backend
2022-02-28 16:00:32,870 INFO    MainThread:626894 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-28 16:00:32,877 INFO    MainThread:626894 [backend.py:ensure_launched():216] starting backend process...
2022-02-28 16:00:32,880 INFO    MainThread:626894 [backend.py:ensure_launched():222] started backend process with pid: 626903
2022-02-28 16:00:32,881 INFO    MainThread:626894 [wandb_init.py:init():469] backend started and connected
2022-02-28 16:00:32,883 INFO    MainThread:626894 [wandb_init.py:init():533] updated telemetry
2022-02-28 16:00:32,884 INFO    MainThread:626894 [wandb_init.py:init():563] communicating current version
2022-02-28 16:00:33,334 INFO    MainThread:626894 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-28 16:00:33,334 INFO    MainThread:626894 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-28 16:00:33,680 INFO    MainThread:626894 [wandb_init.py:init():606] starting run threads in backend
2022-02-28 16:00:36,334 INFO    MainThread:626894 [wandb_run.py:_console_start():1810] atexit reg
2022-02-28 16:00:36,334 INFO    MainThread:626894 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-28 16:00:36,335 INFO    MainThread:626894 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-28 16:00:36,335 INFO    MainThread:626894 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-28 16:00:36,335 INFO    MainThread:626894 [wandb_init.py:init():633] run started, returning control to user process
2022-02-28 16:00:41,214 INFO    MainThread:626894 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 255
2022-02-28 16:00:41,215 INFO    MainThread:626894 [wandb_run.py:_restore():1752] restore
2022-02-28 16:00:43,850 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 16:00:44,189 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 16:00:44,698 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 6427
}

2022-02-28 16:00:44,800 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 16026
}

2022-02-28 16:00:44,902 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 16026
}

2022-02-28 16:00:45,004 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5172
  total_bytes: 16026
}

2022-02-28 16:00:45,106 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16026
  total_bytes: 16026
}

2022-02-28 16:00:45,208 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16026
  total_bytes: 16026
}

2022-02-28 16:00:45,310 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16026
  total_bytes: 16026
}

2022-02-28 16:00:45,413 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16026
  total_bytes: 16026
}

2022-02-28 16:00:45,515 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16026
  total_bytes: 16026
}

2022-02-28 16:00:45,841 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16026
  total_bytes: 16026
}

2022-02-28 16:00:46,324 INFO    MainThread:626894 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16026
  total_bytes: 16026
}
local_info {
}

2022-02-28 16:00:47,391 INFO    MainThread:626894 [wandb_run.py:_append_history():2130] rendering history
2022-02-28 16:00:47,391 INFO    MainThread:626894 [wandb_run.py:_append_summary():2085] rendering summary
2022-02-28 16:00:47,391 INFO    MainThread:626894 [wandb_run.py:_append_files():2180] logging synced files
